packages/recommend-react/src/useFrequentlyBoughtTogether.ts

```diff
@@ -23,9 +23,11 @@ export function useFrequentlyBoughtTogether<TObject>({
   });
   const { status, setStatus } = useStatus('loading');
   const objectIDs = useStableValue(userObjectIDs);
-  const transformItems = useStableValue(userTransformItems);
   const queryParameters = useStableValue(userQueryParameters);
 
+  const transformItemsRef = useRef(userTransformItems);
+  const transformItems = transformItemsRef.current;
```

Comment:
Can you add some tests by updating the `transformItems` prop? You'll realize this doesn't work.

You need to reassign `transformItemsRef.current` to the `transformItems` prop in an effect, and then use `transformItemsRef.current` in the existing effect.